mklive.sh.in

```diff
@@ -303,7 +305,7 @@ while getopts "a:b:r:c:C:T:Kk:l:i:I:S:s:o:p:v:h" opt; do
         k) KEYMAP="$OPTARG";;
         l) LOCALE="$OPTARG";;
         i) INITRAMFS_COMPRESSION="$OPTARG";;
-        I) INCLUDE_DIRECTORY="$OPTARG";;
+        I) INCLUDE_DIRS="$INCLUDE_DIRS $OPTARG";;
```

Comment:
While I really dislike paths that contain spaces, they do happen somewhat frequently and I think they should probably be handled here.
The separator should perhaps be something that cannot be in a filename?

Reply:
IMO just make this script bash and use an array